src/pudl/metadata/sources.py

```diff
@@ -327,6 +327,75 @@
         "working_partitions": {
             "years": [],  # Not yet working!
         },
+        "contributors": [
+            CONTRIBUTORS["catalyst-cooperative"],
+        ],
+        "keywords": sorted(
+            set(
+                [
+                    "form 2",
+                    "ferc2",
+                ]
+                + KEYWORDS["ferc"]
+                + KEYWORDS["finance"]
+            )
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+    },
+    "ferc6": {
+        "title": "FERC Form 6",
+        "path": "https://www.ferc.gov/general-information-1/oil-industry-forms/form-6-6q-historical-vfp-data",
```

Comment:
If this is going to be a mix of XBRL and DBF data we should probably give a path that points to a top level Form 6 page and not just the historical VisualFoxPro data.

Reply:
Updated these to general homepage